Resync web-platform-tests/pointerevents from upstream

aprotyas opened the pull request:
#### 47350644b6a95cc3eb73ca2a039f7b7613b2293a
<pre>
Resync web-platform-tests/pointerevents from upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=274681">https://bugs.webkit.org/show_bug.cgi?id=274681</a>
<a href="https://rdar.apple.com/128698944">rdar://128698944</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/abcbb64783af5b776a3c5f4e13cf4d8756217830">https://github.com/web-platform-tests/wpt/commit/abcbb64783af5b776a3c5f4e13cf4d8756217830</a>

This patch is a mechanical resync of the pointerevents WPT suite with
some test expectation gardening.

* LayoutTests/imported/w3c/web-platform-tests/pointerevents/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/compat/pointerevent_touch_target_after_pointerdown_target_removed.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/compat/pointerevent_touch_target_after_pointerdown_target_removed.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/compat/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/get-device-properties-uniqueid-from-pointer-event.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/get-device-properties-uniqueid-from-pointer-event.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/pointer-event-has-device-properties-uniqueid-from-pointer-event-init.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/pointer-event-has-device-properties-uniqueid-from-pointer-event-init.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-in-iframe-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-in-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-scrollbar-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-in-iframe-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-in-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-scrollbar-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative.html: Copied from LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended.html.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_mouse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_pen-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_touch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_pen-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative.html: Copied from LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed.html.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_mouse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_pen-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_touch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_pen-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_pointers.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_pointers_mouse-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_pointers_pen-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_rightbutton.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_rightbutton_mouse-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_rightbutton_pen-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_mouse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_mouse-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_mouse-right-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_mouse-right-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_nohover_pointers-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_nohover_pointers.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_pen-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_pen-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_pen-right-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_pen-right-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_touch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_touch-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_touch_and_release_at_got_capture-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_touch_and_release_at_got_capture.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_during_capture-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_during_capture.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_during_capture_mouse-auxclick-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_during_capture_mouse-click-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_for_disconnected_node-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_for_disconnected_node.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_remove_setcapture_node-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_remove_setcapture_node.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_mouse_capture_change_hover.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_reappending_last_over_target-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_reappending_last_over_target.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_removing_last_over_element-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_removing_last_over_element.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_after_pointerup_target_removed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_after_pointerup_target_removed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown_mouse-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown_touch-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_releasepointercapture_events_to_original_target_mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_releasepointercapture_events_to_original_target_pen-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_support.js:
(check_PointerEvent):
(moveToDocument):
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/resources/pointerevent_attributes_hoverable_pointers-iframe.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/pointerevents/deviceproperties/get-device-properties-uniqueid-from-pointer-event.tentative-expected.txt: Added.
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/deviceproperties/get-device-properties-uniqueid-from-pointer-event.tentative-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-in-iframe-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-in-iframe-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_mouse-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_pen-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_mouse-expected.txt:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_pen-expected.txt:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_mouse-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_pen-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_remove_setcapture_node-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_removing_last_over_element-expected.txt: Added.
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/279339@main">https://commits.webkit.org/279339@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/9723be21ae1adaf6751b7d36b47a2ff71e826364

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/53218 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/32557 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/5707 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/56497 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/3944 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/39785 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/3680 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/43142 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/2561 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/55316 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/30577 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/45947 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/24273 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/27530 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/3273 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/2100 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/49303 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/3429 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/58093 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/28362 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/3396 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/50543 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/29579 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/46168 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/49860 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/11601 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/30499 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/29337 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

gsnedders commented on LayoutTests/platform/gtk/TestExpectations:
```diff
@@ -127,6 +123,9 @@ imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-action-svg-none
 imported/w3c/web-platform-tests/pointerevents/pointerevent_sequence_at_implicit_release_on_click.html [ Skip ]
 
+# webkit.org/b/274686 -- Crashing under PointerCaptureController::processPendingPointerCapture
+imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html [ Crash ]
```
```suggestion
webkit.org/b/274686 imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html [ Crash ]
```

aprotyas replied:
Applied this suggestion and moved the expectation to `glib/TestExpectations` instead, https://github.com/WebKit/WebKit/commit/1584a86e9189548c9b5967d66cbab2d00ff6cd67.

gsnedders commented on LayoutTests/platform/mac-wk1/TestExpectations:
```diff
@@ -2308,6 +2306,9 @@ imported/w3c/web-platform-tests/pointerevents/pointerevent_mouse_capture_change_
 imported/w3c/web-platform-tests/pointerevents/pointerevent_pointercapture-in-custom-element.html [ Pass Failure ]
 
+# webkit.org/b/274686 -- Crashing under PointerCaptureController::processPendingPointerCapture
+imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html [ Crash ]
```
This is redundant when you also have an expectation for this test in `mac`.

aprotyas replied:
True. I'll remove this.

aprotyas replied:
Removed in https://github.com/WebKit/WebKit/commit/1584a86e9189548c9b5967d66cbab2d00ff6cd67.

gsnedders commented on LayoutTests/platform/wpe/TestExpectations:
```diff
@@ -148,6 +144,9 @@ imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-adjustment_clic
 imported/w3c/web-platform-tests/pointerevents/predicted_events_attributes.html?touch [ Skip ]
 
+# webkit.org/b/274686 -- Crashing under PointerCaptureController::processPendingPointerCapture
+imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html [ Crash ]
```
Given both gtk and wpe have this expectation, this should just be in glib rather than duplicated.

Or maybe, given this is in glib and mac, we should just have this at the top-level TestExpectations and a PASS expectation in ios?

aprotyas replied:
This is glib and mac mainly because the entire suite is skipped on iOS: https://github.com/WebKit/WebKit/blob/60717e784729ccc4d0239f8027f09ecebf09237d/LayoutTests/platform/ios/TestExpectations#L6584

aprotyas replied:
Unfortunately I can't place this expectation in the top-level TestExpectations because the more platform-specific blanket PASS expectation in `LayoutTests/platform/mac/TestExpectations` would override it! (I tried!)

https://github.com/WebKit/WebKit/blob/60717e784729ccc4d0239f8027f09ecebf09237d/LayoutTests/platform/mac/TestExpectations#L93

aprotyas replied:
Removed the `wpe`/`gtk` duplicate expectations and moved to `glib/TestExpectations` instead. https://github.com/WebKit/WebKit/commit/1584a86e9189548c9b5967d66cbab2d00ff6cd67

gsnedders commented on LayoutTests/platform/mac/TestExpectations:
```diff
@@ -1485,6 +1480,9 @@ imported/w3c/web-platform-tests/pointerevents/pointerup_after_pointerdown_target
 imported/w3c/web-platform-tests/pointerevents/touch-action-with-swipe-dir-change.html?touch [ Skip ]
 
+# webkit.org/b/274686 -- Crashing under PointerCaptureController::processPendingPointerCapture
+imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html [ Crash ]
```
```suggestion
webkit.org/b/274686 imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html [ Crash ]
```

aprotyas replied:
Fixed in https://github.com/WebKit/WebKit/commit/1584a86e9189548c9b5967d66cbab2d00ff6cd67

gsnedders reviewed (commented):
The TestExpectations changes seem somewhat redundant.

We also generally avoid CRASH expectations because they're slow to process, and we generally know when we fix them, preferring SKIP instead.

aprotyas committed:
Resync web-platform-tests/pointerevents from upstream
https://bugs.webkit.org/show_bug.cgi?id=274681
rdar://128698944

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/abcbb64783af5b776a3c5f4e13cf4d8756217830

This patch is a mechanical resync of the pointerevents WPT suite with
some test expectation gardening.

* LayoutTests/imported/w3c/web-platform-tests/pointerevents/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/compat/pointerevent_touch_target_after_pointerdown_target_removed.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/compat/pointerevent_touch_target_after_pointerdown_target_removed.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/compat/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/get-device-properties-uniqueid-from-pointer-event.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/get-device-properties-uniqueid-from-pointer-event.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/pointer-event-has-device-properties-uniqueid-from-pointer-event-init.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/pointer-event-has-device-properties-uniqueid-from-pointer-event-init.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/deviceproperties/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-in-iframe-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-in-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-scrollbar-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-in-iframe-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-in-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-scrollbar-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative.html: Copied from LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended.html.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_mouse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_pen-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_touch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_pen-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative.html: Copied from LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed.html.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_mouse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_pen-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_touch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_pen-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_pointers.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_pointers_mouse-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_pointers_pen-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_rightbutton.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_rightbutton_mouse-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_hoverable_rightbutton_pen-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_mouse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_mouse-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_mouse-right-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_mouse-right-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_nohover_pointers-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_nohover_pointers.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_pen-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_pen-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_pen-right-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_pen-right-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_touch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_attributes_touch-nonstandard-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_touch_and_release_at_got_capture-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_touch_and_release_at_got_capture.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_during_capture-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_during_capture.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_during_capture_mouse-auxclick-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_during_capture_mouse-click-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_for_disconnected_node-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_for_disconnected_node.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_remove_setcapture_node-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_remove_setcapture_node.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_mouse_capture_change_hover.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_reappending_last_over_target-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_reappending_last_over_target.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_removing_last_over_element-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_removing_last_over_element.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_after_pointerup_target_removed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_after_pointerup_target_removed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown_mouse-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown_touch-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointermove_isprimary_same_as_pointerdown-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_releasepointercapture_events_to_original_target_mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_releasepointercapture_events_to_original_target_pen-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_support.js:
(check_PointerEvent):
(moveToDocument):
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/resources/pointerevent_attributes_hoverable_pointers-iframe.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/pointerevents/deviceproperties/get-device-properties-uniqueid-from-pointer-event.tentative-expected.txt: Added.
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/deviceproperties/get-device-properties-uniqueid-from-pointer-event.tentative-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-in-iframe-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointer-events-none-skip-scroll-will-change-in-iframe-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_mouse-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_interleaved.tentative_pen-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_mouse-expected.txt:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_appended_pen-expected.txt:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_mouse-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_after_target_removed_interleaved.tentative_pen-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_lostpointercapture_remove_setcapture_node-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_removing_last_over_element-expected.txt: Added.
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:

Canonical link: https://commits.webkit.org/279339@main
